src/screens/Analytics/Logs/LogUtils/AuditLogUI.res

```diff
@@ -10,7 +10,7 @@ module LogDetailsSection = {
       }
     }
 
-    <div className="border-b-2 border-border-light-grey pb-3 px-5 py-3">
+    <div className="border-border-light-grey pb-3 px-5 py-3">
```

Comment:
border-border-light-grey is this a right class name ?

Reply:
corrected